src/web/components/Core/Form.tsx

```diff
@@ -52,10 +52,16 @@ export function Form<T extends FieldValues>({
         await onSubmit(formData);
       } catch (err) {
         if (onError) onError(err);
-        const message =
-          axios.isAxiosError(err) && (err.response?.data ?? [])[0]?.message
-            ? ((err.response?.data ?? [])[0]?.message as string)
-            : 'Something went wrong, please try again';
+        let message = 'Something went wrong, please try again';
+        if (axios.isAxiosError(err)) {
+          if (err.response?.data?.message) {
```

Comment:
So I'm going to leave this here for now because I know that it works.  The bigger issue is that I couldn't actually use Form.tsx here because it didn't support most of the hooks that I needed, so I had to use the vanilla FormProvider.  Unfortunately now my error messages aren't displaying properly on the form.  I'm going to write a new ticket for this.

Reply:
So I moved this to errorHelpers and in the process realized that my current implementation isn't even using this because I'm no longer using Form due to limitations.  I updated my dialog to show errors properly and I'm logging a ticket to update the Form component to add more react form hooks.